File: p001/p1.go
// Project Euler Problem #1

package main

import "fmt"

func main() {
	sum := 0
	for i := 0; i < 1000; i++ {
		if i%3 == 0 || i%5 ==0 {
			sum += i
		}
	}
	fmt.Println("The sum of all natural numbers <1000 and divisible by 3 and 5 is:", sum)
}

File: p001/p1.rs
// Project Euler Problem #1

fn main() {

	let mut sum:i32 = 0;
	let mut i:i32 = 1;

	while 3*i < 1000 {
		sum += 3*i; 
		i+=1;
	}

	i = 1;
	
	while 5*i < 1000 {
		if 5*i % 3 != 0{
		    sum += 5*i;
		}
		i+=1;
    } 
 	println!("The sum of all natural numbers <1000 and divisible by 3 or 5 is: {}", sum);
}
